module/importer/src/Infrastructure/Action/GroupingProductImportAction.php

```diff
@@ -21,15 +24,24 @@ public function action(
         string $template,
         array $categories,
         array $children,
-        array $attributes = []
+        array $attributes = [],
+        ImportId $importId = null
```

Comment:
As far as I am concerned, this is not exactly a good solution,
records are added because of a specific need !
Leaving it like that will stay in the project permanently and you will have to remember to maintain it. 
Probably eventually it should be added in each action and as a mandatory value. Only then we change the interface ... 
Maybe it would be worth to think how to do it.